build_rules.gradle

```diff
@@ -321,7 +321,6 @@ ext.getJavaRelocatedPath = { String suffix ->
 
 ext.DEFAULT_SHADOW_CLOSURE = {
```

Comment:
Add a link to the https://github.com/johnrengelman/shadow/blob/98191096a94674245c7b3e63975df9e14f67074e/src/main/groovy/com/github/jengelman/gradle/plugins/shadow/internal/DefaultDependencyFilter.groovy#L123 with an explanation of how include/exclude matching works.